distribution/tools/entitlement-runtime/src/test/resources/org/elasticsearch/entitlement/runtime/policy/test-policy.yaml

```diff
@@ -0,0 +1,7 @@
+entitlement-module-1:
```

Comment:
nit: maybe make this "example-module-name", with the `1` it looks like this is an arbitrary id

Reply:
Changed.